package.json

```diff
@@ -5,12 +5,12 @@
   "repository": "prettier/prettier-php",
   "author": "Lucas Azzola <@azz>",
   "license": "MIT",
-  "type": "module",
   "unpkg": "./standalone.js",
   "browser": "./standalone.js",
   "exports": {
     ".": {
       "browser": "./standalone.js",
+      "require": "./standalone.js",
```

Comment:
Just in case we need ship Commonjs version for N ode.js.

Let's revert this, make the node.js version ESM only for now. 

User have to explicitly `require()` the `standalone` entry.

Reply:
@fisker thanks.
I added this line while testing this PR with [prettier-vscode](https://github.com/prettier/prettier-vscode) on VSCode because I was unable to resolve @prettier/plugin-php and can not run formatting.

```php
["INFO" - 3:37:57 AM] Using config file at '/home/shuhei/develop/plugin-php-test/.prettierrc.json'
["ERROR" - 3:37:57 AM] Error handling text editor change
["ERROR" - 3:37:57 AM] Error resolve node module '@prettier/plugin-php'
Error: Error resolve node module '@prettier/plugin-php'
    at n (/home/shuhei/.vscode/extensions/esbenp.prettier-vscode-10.1.0/dist/extension.js:1:2718)
    at /home/shuhei/.vscode/extensions/esbenp.prettier-vscode-10.1.0/dist/extension.js:1:3068
    at Array.map (<anonymous>)
    at t.resolveConfigPlugins (/home/shuhei/.vscode/extensions/esbenp.prettier-vscode-10.1.0/dist/extension.js:1:3003)
    at t.ModuleResolver.resolveConfig (/home/shuhei/.vscode/extensions/esbenp.prettier-vscode-10.1.0/dist/extension.js:1:7290)
    at t.default.getSelectors (/home/shuhei/.vscode/extensions/esbenp.prettier-vscode-10.1.0/dist/extension.js:1:11562)
    at t.default.handleActiveTextEditorChanged (/home/shuhei/.vscode/extensions/esbenp.prettier-vscode-10.1.0/dist/extension.js:1:10924)
```

Should I revert this line and look for another solution?

Reply:
Sorry, I didn't expect you read prettier config file.

I guess there is no good solution. Let me think about it..

Reply:
I misunderstand this agian, it's a different case. It should be solved in `prettier-vscode` side. https://github.com/prettier/prettier-vscode/issues/3104